Kernel/TTY/TTY.cpp

```diff
@@ -155,6 +155,9 @@ void TTY::emit(u8 ch)
         if (ch == m_termios.c_cc[VSUSP]) {
             dbg() << tty_name() << ": VSUSP pressed!";
             generate_signal(SIGTSTP);
+            auto process = Process::from_pid(m_pgid);
```

Comment:
What if this returns `nullptr`

Reply:
_can_ that return `nullptr`?

AFAIK the pgid has to be a valid pid?

Reply:
Valid when setting yeah. But just looking at this I get suspicious that maybe we're not clearing `m_pgid` when a process dies.

Reply:
Makes sense.